cloud/aws/variables.tf

```diff
@@ -6,6 +6,11 @@ variable "cockroach_port" {
   default = "26257"
 }
 
+# Comma-separated list of directories added as auxiliary stores.
+variable "aux_data_dirs" {
+  default = ""
```

Comment:
I'm not convinced what for, all that gives us is more ways of screwing things up and more to check. There always needs to be at least one store. What's the part about `formatlist` about?